src/test/resources/toStringBugTest/TestClass2.java

```diff
@@ -2108,37 +2108,7 @@ public Page<Redemption> searchRedemptionsForPay(Long rdmMerchantNo, Long userNo)
     }
 
     private APIErrorCode getRedemptionErrorCode(int status) {
-
```

Comment:
Please don't modify code that is used as test input. There is little benefit as it isn't actually run, but adds significant overhead to the review process as we need to verify that the involved test still does what it is supposed to do.

The other change looks clean